internal/presenters/presenter_sarif_results_pretty.go

```diff
@@ -0,0 +1,208 @@
+package presenters
+
+import (
+	"bytes"
+	"fmt"
+	"slices"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/snyk/code-client-go/sarif"
+
+	"github.com/snyk/go-application-framework/pkg/local_workflows/code_workflow"
+	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
+)
+
+type Finding struct {
+	ID            string
+	Severity      string
+	SeverityLevel int
+	Title         string
+	Message       string
+	Path          string
+	Line          int
+}
+
+type FindingsSummary struct {
+	High   int
+	Medium int
+	Low    int
+}
+
+func convertSarifToFindingsList(input sarif.SarifDocument) []Finding {
+	var findings []Finding
+	for _, run := range input.Runs {
+		for _, result := range run.Results {
+			severity := "n/a"
+			severityLevel := 0
+			var rule sarif.Rule
+			for _, ruleItem := range run.Tool.Driver.Rules {
+				if ruleItem.ID == result.RuleID {
+					rule = ruleItem
+					break
+				}
+			}
+
+			if result.Level == "note" {
```

Comment:
Suggestion: this is a bit of redundant logic, the conversion from sarif level to severity is already available and the ordering is also available through the test summary. Let's remove this code and keep a single source of thruth.